kubectl-cloudflow/cmd/deploy_command.go

```diff
@@ -301,16 +301,14 @@ func dockerConfigEntryExists(k8sClient *kubernetes.Clientset, namespace string,
 	return false
 }
 
-func createOrUpdateStreamletSecrets(k8sClient *kubernetes.Clientset, namespace string, streamletNameSecretMap map[string]*corev1.Secret) {
-	for streamletName, secret := range streamletNameSecretMap {
-		if _, err := k8sClient.CoreV1().Secrets(secret.ObjectMeta.Namespace).Get(secret.ObjectMeta.Name, metav1.GetOptions{}); err != nil {
-			if _, err := k8sClient.CoreV1().Secrets(namespace).Create(secret); err != nil {
-				printutil.LogAndExit("Failed to create secret %s, %s", streamletName, err.Error())
-			}
-		} else {
-			if _, err := k8sClient.CoreV1().Secrets(namespace).Update(secret); err != nil {
-				printutil.LogAndExit("Failed to update secret %s, %s", streamletName, err.Error())
-			}
+func createOrUpdateAppInputSecret(k8sClient *kubernetes.Clientset, namespace string, appInputSecret *corev1.Secret) {
+	if _, err := k8sClient.CoreV1().Secrets(appInputSecret.ObjectMeta.Namespace).Get(appInputSecret.ObjectMeta.Name, metav1.GetOptions{}); err != nil {
```

Comment:
Minor: ObjectMeta is not needed. "appInputSecret.Namespace" and "appInputSecret.Name" will work.